Workbooks/Windows Virtual Desktop/AtScale/User Report/User Report.workbook

```diff
@@ -1562,6 +1218,346 @@
+                    "query": "WVDFeeds\r\n| where UserName == \"{UserName}\" and (\"{SelectedClientType}\" == \"*\" or ClientType == \"{SelectedClientType}\")\r\n| extend ClientTypeShort=replace(\"com.microsoft.rdc.\", \"\", ClientType)\r\n| make-series FeedRefreshes=dcount(CorrelationId) on TimeGenerated from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by strcat(ClientTypeShort, \" v\", ClientVersion)\r\n",
+                    "size": 0,
+                    "title": "Feed refreshes by client and version",
+                    "timeContextFromParameter": "TimeRange",
+                    "showExportToExcel": true,
+                    "queryType": 0,
+                    "resourceType": "microsoft.desktopvirtualization/workspaces",
+                    "crossComponentResources": [
+                      "{wvdworkspaces}"
+                    ],
+                    "visualization": "areachart",
+                    "gridSettings": {
+                      "formatters": [
+                        {
+                          "columnMatch": "ClientType",
+                          "formatter": 5
+                        },
+                        {
+                          "columnMatch": "ClientVersion",
+                          "formatter": 5
+                        },
+                        {
+                          "columnMatch": "LastSeen",
+                          "formatter": 6,
+                          "formatOptions": {
+                            "aggregation": "Max",
+                            "customColumnWidthSetting": "30ch"
+                          }
+                        },
+                        {
+                          "columnMatch": "FeedRefreshes",
+                          "formatter": 4,
+                          "formatOptions": {
+                            "min": 0,
+                            "palette": "gray",
+                            "aggregation": "Sum",
+                            "customColumnWidthSetting": "128px"
+                          }
+                        },
+                        {
+                          "columnMatch": "Connections",
+                          "formatter": 4,
+                          "formatOptions": {
+                            "min": 0,
+                            "palette": "blue",
+                            "aggregation": "Sum",
+                            "customColumnWidthSetting": "115px"
+                          }
+                        }
+                      ],
+                      "hierarchySettings": {
+                        "treeType": 1,
+                        "groupBy": [
+                          "ClientType"
+                        ],
+                        "expandTopLevel": true,
+                        "finalBy": "ClientVersion"
+                      }
+                    },
+                    "sortBy": []
+                  },
+                  "customWidth": "42",
```

Comment:
I think we wanted these (at scale and scoped) to be 100% width, right?

Reply:
Updated. It makes sense to set the width to 100%.